### PR TITLE
Changing node traffic interface

### PR DESCRIPTION
**Pull request by tpoitras**
https://bugzilla.redhat.com/show_bug.cgi?id=1301654

This has undergone tech review updates already (see BZ)

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -426,3 +426,38 @@ installation] using the `*openshift_node_kubelet_args*` variable. For example:
...
+Although  `*openshift_node_set_node_ip*` can be useful as a workaround for the
+cases stated in this section, it is generally not suited for production
+environments. This is because the node will no longer function properly if it
+receives a new IP address.
+====
+
+Changing the node traffic interface may be needed in cases where:
+
+- {product-title} is installed in a cloud provider where internal hostnames are not configured/resolvable by all hosts.
+- The node's IP from the master's perspective is not the same as the node's IP from its own perspective.
```
Comment (bfallonf) [inline admin_guide/manage_nodes.adoc]: I'd move these cases to the top of the section.

**Comment by bfallonf** [comment]
@tpoitras Just one comment from me.

**Comment by tpoitras** [comment]
Thanks, @bfallonf ! :+1:

**Comment by tpoitras** [comment]
[rev_history]
|xref:../admin_guide/manage_nodes.adoc#manage-node-change-node-traffic-interface[Cluster Administration -> Managing Nodes]
|Added details on how to change the node traffic interface.
%

**Pull request merged**